config/vendor_api/draft.yml

```diff
@@ -16,6 +16,52 @@ servers:
 - description: Production
   url: https://www.apply-for-teacher-training.service.gov.uk/api/v1.1
 paths:
+  "/applications":
+    get:
+      tags:
+      - Application management
+      summary: Get many applications
+      description: |
+        This endpoint can be used to retrieve applications for the authenticating
+        provider. Applications are returned with the most recently updated ones first.
+
+        Use the `since` parameter to limit the number of results. This is intended
+        to make it possible to check for new or updated applications regularly.
+      parameters:
+      - name: since
+        description: Include only applications changed or created on or since a date
+          and time. Times should be in ISO 8601 format.
+        in: query
+        required: true
+        example: 2019-12-13T12:00:00Z
+        schema:
+          type: string
+          format: date-time
+      - name: page
+        description: Page number
+        in: query
+        required: false
+        example: 1
+        schema:
+          type: integer
+      - name: per_page
+        description: Number of records to return per page (e.g. 50)
+        in: query
+        required: false
+        example: 100
+        schema:
+          type: integer
+      responses:
+        '200':
+          description: An array of applications
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/MultipleApplicationsResponse"
+        '401':
+          "$ref": "#/components/responses/Unauthorized"
+        '422':
+          "$ref": "#/components/responses/ParameterMissing"
```

Comment:
could you also add the links and meta sections back please?

Reply:
maybe copy 1.1 before the last PR merge so we don't miss anything else?

Reply:
Apologies yes, I've copied the previous v1.1.yml to draft as mentioned and removed anything extraneous. We should have links/meta definitions and pagination params now.